Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -841,7 +829,7 @@ public class DynamicHlsController : BaseJellyfinApiController
             [FromRoute, Required] Guid itemId,
             [FromRoute, Required] string playlistId,
             [FromRoute, Required] int segmentId,
-            [FromRoute, Required] string container,
```

Comment:
But it's not in the url (see last post) - so it should either be 
```
[FromRoute] string? container 
```
or removed as it will be picked up later in the helper.

Reply:
The route I see is `[HttpGet("Videos/{itemId}/hls1/{playlistId}/{segmentId}.{container}")]`

Reply:
but if it's a nullable string then HlsVideoRequestDto needs also to be nullable which it isn't.

Reply:
I'm hitting the route.

[HttpGet("Videos/{itemId}/master.m3u8")]
[HttpHead("Videos/{itemId}/master.m3u8", Name = "HeadMasterHlsVideoPlaylist")]

Reply:
I suspect it will be the same for all the main.m3u8 routes.       

```
{"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"|eec14152-49c2fadad79d0f06.","errors":{"container":["The container field is required."]}}
```

Reply:
In the audio playlist route (below the main.m3u8 route L340~) it gets it FromQuery, maybe that is fine too if we want to support clients selecting an explicit container for the playlist.

Reply:
Alright so after a bunch of testing and stepping. All m3u8 routes (both audio and video) just need to container parameter removed. (Just pass string.Empty into the Dto)

Reply:
I don't think any of the HLS playlist routes can be something other than m3u8, so removing it fully is the way to go, imo.

Reply:
Updated code with containers removed for .m3u8 routes.

Reply:
I think this parameter still needs to be `Required` on this endpoint. The path it is listening to `Videos/{itemId}/hls1/{playlistId}/{segmentId}.{container}` won't work without it.

Reply:
Yes - i left that one in as it has the container profile.